### PR TITLE
Fixing some typos and grammar issues

### PR DESCRIPTION
**Pull request by noahfalk**
I'm contemplating some other adjustments to content that might add more conceptual information, but that would be later if it happens. This PR is for the small grammatical stuff.

### CODE REVIEW

File: docs/core/diagnostics/distributed-tracing.md [hunk 1]
```diff
@@ -59,18 +59,18 @@ Here is example for how to listen to the Activity events:
...
 
 > [!NOTE]
-> To access all added .NET 5.0 capabilities, ensure referencing the [System.Diagnostics.DiagnosticSource](https://www.nuget.org/packages/System.Diagnostics.DiagnosticSource/) NuGet package version 5.0 or later. This package can be used in libraries and apps targeting any supported version of the .NET Framework, .NET Core, and .NET Standard. If targeting .NET 5.0 or later, no need to manually reference the package as it is included in the shared library installed with the .NET SDK.
+> To access all added .NET 5.0 capabilities, ensure your project references the [System.Diagnostics.DiagnosticSource](https://www.nuget.org/packages/System.Diagnostics.DiagnosticSource/) NuGet package version 5.0 or later. This package can be used in libraries and apps targeting any supported version of the .NET Framework, .NET Core, and .NET Standard. If targeting .NET 5.0 or later, there is no need to manually reference the package as it is included in the shared library installed with the .NET Runtime.
```
Comment (tarekgh) [inline docs/core/diagnostics/distributed-tracing.md:65]: should it be `shared libraries` instead of `shared library`?

**Review by tarekgh — APPROVED** [review]
Thanks @noahfalk for fixing my grammar mistakes :-)

**Review by sywhang — APPROVED** [review]
would be nice to add xref links to the API docs for ActivitySource/Activity, but this looks good nonetheless :)

File: docs/core/diagnostics/distributed-tracing.md (hunk 1 above)
Comment (noahfalk) [reply docs/core/diagnostics/distributed-tracing.md:65]: I think both are valid interpretations. Sometimes we refer to BCL conceptually as a single shared library, as in shared library == TFM. Other times we specifically mean shared library == assembly so the BCL is 'shared libraries'.

**Pull request merged**